MetasysServicesExampleApp/Program.cs

```diff
@@ -343,6 +349,67 @@ static void Main(string[] args)
                 Console.WriteLine($"\n Id: {item.Id}, Name: {item.Name}, ItemReference: {item.ItemReference}");
             }
 
+            Console.WriteLine("Enter object id to get alarm details: ");
+            string objectId = Console.ReadLine();
+            
+            Console.WriteLine(string.Format("\n Alarm details found for this object {0}", objectId));
+
+            string getAlarmsForObject;
+            AlarmFilter alarmFilterForObject = new AlarmFilter();
+
+            if (args.Length != 12)
+            {
+                Console.WriteLine("Please enter these parameters separated by space: Start Time, End Time, Priority range, Type, Exclude pending, Exclude acknowledged, Exclude discarded, Attribute, Category, Page, Page Size, Sort" +
+                    "\nRefer to the metasys-server/basic-services-dotnet README if you need help getting started.");
+                getAlarmsForObject = Console.ReadLine();
+                args = getAlarmsForObject.Split(' ');
+

```

Comment:
The if condition should be here once you read value into args and then check args.Length != null